### DISCUSSION TITLE
Work as a linter

### DISCUSSION THREAD

**Discussion by njfamirm** [ianyh/Amethyst#1675] [Q&A] [answered]
Hey everyone!

It is possible use this app as linter of the windows?
I'm like this app, but I'm want this just when i want, not everytime...

It's possible to use this app without automatic align and sort windows? just when when enter shortcut?

**Comment by ianyh** [comment]
It sounds like what you want isn't an automatic tiling window manager. There are plenty of others that allow you to organize on demand. You could probably make it work with Amethyst, but it would be kind of kludgy.

**Reply by njfamirm** [reply]
> There are plenty of others that allow you to organize on demand

Which?

**Reply by ianyh** [reply, marked as answer]
I'm not super up to date on other window managers, but Rectangle comes to mind.

**Reply by njfamirm** [reply]
thanks
